packages/report-server/src/reportBuilder/fetch/fetch.ts

```diff
@@ -25,11 +25,13 @@ const buildParams = (params: unknown): FetchParams => {
     throw new Error(`Expected object but got ${params}`);
   }
 
-  if (Object.keys(params).length > 1) {
+  const { aggregation, ...fetchParams } = params;
+
+  if (Object.keys(fetchParams).length > 1) {
```

Comment:
Truth be told I'm not at all happy with the validation going on here, however I've been meaning to re-write report-server to use the new server-boilerplate and once that's been done I think we can take a fresh look at the validation logic...